Number_types/include/CGAL/boost_mp.h

```diff
@@ -192,24 +194,41 @@ struct RET_boost_mp_base
 
     struct To_interval
         : public CGAL::cpp98::unary_function< Type, std::pair< double, double > > {
+
         std::pair<double, double>
         operator()(const Type& x) const {
```

Comment:
Do we get an efficient `to_interval(cpp_int)` (not relying on `convert_to<double>`+`compare`+`nextafter`) with this patch? You have probably written one somewhere, but I don't see it. On the other hand, this is less important than rationals.

Reply:
Actually no. I have it only for rational types: `cpp_rational` and `Quotient<cpp_int>`.

Reply:
`get_interval_exp` is probably just one `ldexp` away, if you feel like adding it.